http/src/main/resources/init-resources/azure_vm_init_script.sh

```diff
@@ -167,6 +167,15 @@ echo "VALID_HOSTS = ${VALID_HOSTS}"
 
 /anaconda/envs/py38_default/bin/pip3 install seaborn==0.13.0
 
+# Wait for lock to resolve before apt install
+
```

Comment:
Nit: Could you link the article you mentioned during standup that explains the issue?